packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -1,18 +1,19 @@
 // @ts-check
```

Comment:
> We had to transform Popup.jsx and OAuthWindow.jsx into functions to
avoid to be blocked by browser when opening a popup (Too many useEffect
with render tend to block the popup)

Can you explain me why converting to function has changed that fact? Since you talk about `useEffect` I was expecting that popup & oauthwindow were already in that form.

Reply:
When I first tried to avoid unmounting OAuthWindow when the account was removed, I tried to put the OAuthWindow component in a provider, like I do with the confirm dialog and the OAuthWindow component would be piloted with a service. It seems the this way introduced too many step between the user action and the window.open command, which lead to a popup blocking by the browser.

We needed to directly call window.open as soon as possible to not be blocked